### PR TITLE
New category css

### PR DESCRIPTION
**Pull request by mikedao**
Improved CSS for new category for platform admin.

### CODE REVIEW

File: app/views/admin/categories/index.html.erb [hunk 1]
```diff
@@ -10,8 +10,8 @@
     <div class="category">
       <h4 class="form-title"> Create New Category </h4>
       <%= form_for(:categories, placeholder: "category",  url: admin_categories_path) do |f| %>
-        <%= f.text_field :name %>
-        <p><%= f.submit "Add Category" %></p>
+        <p><%= f.text_field :name, class: "form-control"  %></p>
+        <p class="text-center"><%= f.submit "Add Category", class: "btn btn-info" %></p>
```
Comment (EmilyMB) [inline app/views/admin/categories/index.html.erb]: The class name "btn-info" seems odd for a submit button...is that standard?

File: config/routes.rb [hunk 2]
```diff
@@ -8,8 +8,6 @@
   resources :categories, only: [:show, :index]
   resources :projects
 
-  get "/choose", to: "static_pages#choose"
```
Comment (EmilyMB) [inline config/routes.rb]: :+1:

File: app/views/admin/categories/index.html.erb (hunk 1 above)
Comment (mikedao) [inline app/views/admin/categories/index.html.erb]: the info controls color

File: config/routes.rb (hunk 2 above)
Comment (mikedao) [inline config/routes.rb]: :sparkles:

**Pull request merged**